docs/discover/document-explorer.asciidoc

```diff
@@ -0,0 +1,90 @@
+[[document-explorer]]
+== Document Explorer
+
+beta::[]
+
+*Discover* has a new *Document Explorer* with resizable columns, better data sorting and comparison,
+and a full screen view.
+
+[role="screenshot"]
+image::images/document-explorer.png[Document Explorer with improved look over classic view]
+
+To use the *Document Explorer* instead of the classic document table:
+
+. Open *Discover*.
+. In the toolbar, click *Options*, and then click *Try Document Explorer*.
+. Turn off the classic view and save the changes.
+
+[float]
+[[document-explorer-columns]]
+=== Reorder and resize columns
+
+Reorder and resize columns in the *Document Explorer* with drag and drop.
```

Comment:
Instead of restating the title, maybe include the value prop? Why would users want to reorder and resize their columns? Better organization? Better view of their data?